### PR TITLE
[Issue 426] update issue templates new

### LINKED ISSUE #426: [Task]: Update issue templates with new hierarchy and labels
### Summary

Now that the group has settled on the following issue hierarchy: `milestone > epic > issue` we need to update our existing issue templates to reflect this new hierarchy and update the default labels.

### Dependencies

- [x] #320 

### Specification

Update the list of issue templates to reflect this list

1. Bug Report
    - Sections: No change
    - Labels: `bug`
2. Feature Request
    - Sections: No change
    - Labels: `enhancement`
3. Internal - 30k
    - Sections
      - Description
      - Business goals
      - Dependencies
      - Functionality
      - Definition of Done
    - Labels: `deliverable: 30k ft`
4. Internal - 10k
    - Sections
      - Definition
      - Business goals
      - In scope
      - Out of scope
      - Tasks
      - Definition of Done
    - Labels: `deliverable: 10k ft`
5. Internal - Task
    - Sections: No Change
    - Labels: 
6. Internal - ADR
    - Sections: No Change
    - Labels: `docs: adr`
7. Internal - Deliverable spec
    - Sections: No Change
    - Labels: `docs: deliverable`

### Acceptance criteria

- [x] The issue templates have been updated to reflect the requirements listed above
- [x] The project automations use the new labels to automatically add tickets to a project

### PR DESCRIPTION
**Pull request by widal001**
## Summary
Fixes #426 

### Time to review: __5 mins__

- Added two new templates for product tickets:
  - Internal - 30k deliverable (Note: this is a markdown template to work with `/` commands)
  - Internal - 10k deliverable (Note: this is a markdown template to work with `/` commands)
- Updated the language and sections in our existing templates:
  - Internal - ADR
  - Internal - Task
  - Bug report
  - Feature request

To see what the new issue templates look like, [check out this example](https://github.com/widal001/project-demo/issues/new/choose)

Also the reason I chose to use markdown templates for 30k and 10k deliverable tickets was to enable us to [use slash commands ](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-slash-commands#using-slash-commands)`/` to populate a blank issue with the template.

Screenshot 2023-11-16 at 12 11 38 PM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/internal_task.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-name: Task (Internal Use Only)
-description: Tracking our work through tasks
+name: Internal - Task
+description: Describes an individual task that needs to be completed
 title: "[Task]: "
+labels: ["project: grants.gov"]
```
Comment (acouch) [inline .github/ISSUE_TEMPLATE/internal_task.yml:4]: Thanks for adding this, it is tedious of course to add every time. I imagine we could revisit in the future if necessary if other projects need to use the internal task.

**Review by acouch — APPROVED** [review]
Great update, thanks!

**Review by sarahknoppA6 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/internal_task.yml (hunk 1 above)
Comment (sumiat) [reply .github/ISSUE_TEMPLATE/internal_task.yml:4]: +++

**Review by sumiat — APPROVED** [review]
Thanks for doing this!

**Review by margaretspring — COMMENTED** [review]
Will be good to have less!

**Pull request merged**